src/game/utils.js

```diff
@@ -38,4 +38,13 @@ export const areAllConnected = points => {
     points = points.filter(isContained(current));
   }
   return points.length === 0;
-}
\ No newline at end of file
+}
+
+export const calculateVector = (source, destination) => {
```

Comment:
Rename to subtractVector perhaps?

Reply:
This doesn't return the result of a createPoint, unlike the function below.

Reply:
But... a vector isn't a point? 😜